app/src/main/java/org/torproject/android/OrbotActivity.kt

```diff
@@ -262,12 +268,6 @@ class OrbotActivity : BaseActivity() {
         super.onActivityResult(requestCode, resultCode, data)
         if (requestCode == REQUEST_CODE_VPN && resultCode == RESULT_OK) {
             connectViewModel.triggerStartTorAndVpn()
-        } else if (requestCode == REQUEST_CODE_SETTINGS && resultCode == RESULT_OK) {
-            Prefs.defaultLocale = data?.getStringExtra("locale") ?: Locale.getDefault().language
```

Comment:
this logic, which only ran on legacy androids when locale changes were made in the settings app, can now just be handled in the settings fragment